### ISSUE TITLE
Change "DM" and "Group Messages" icons to be readable by mere mortals

### ISSUE THREAD

**Issue by williamkray** [vector-im/element-android#2492]
**Is your feature request related to a problem? Please describe.**
Currently the icons at the bottom of the screen for 1:1 chats vs. group rooms includes:
1. a speech bubble for 1:1 DMs
2. a circle with `#` inside

Users who are unfamiliar with IRC or Slack or other more technically-focused conversation platforms have no recognition of the relevance of the `#` symbol equating to a chatroom, and visually these two icons do a poor job of demonstrating the differences between these two kinds of conversation.

**Describe the solution you'd like**
The solution would be to change these icons to represent:
1. a single person for 1:1 DM rooms
2. a group of three people for multi-participant rooms (see https://static.thenounproject.com/png/446003-200.png for a simple representation of this)

**Describe alternatives you've considered**
A more advanced function might be to allow users to configure which set of icons they want within the application, although this seems unnecessary if the icons are in-line with the design style of the application and _do a reasonable job of representing what they are for_

**Comment by notramo** [comment]
I think the chat bubble is good icon for DMs.
Educating the users isn't bad. Thus, the # icon is good for highlighting the difference between Matrix and other platforms. But they don't know at first what that button means.
**What about having a design similar to Facebook Messenger?** It has an icon, and text below it. So the buttons in Element should have the text below the icons **Chats** for DMs, and **Groups** for group rooms.

**Comment by pythonInRelay** [comment]
Yes please to both of these. I think it's also more "normal" to have the chat bubble icon and # at the top too.
